main/webapp/modules/core/langs/translation-es.json

```diff
@@ -455,18 +455,18 @@
 		"by-judg": "Por parámetro",
 		"best-score": "Por puntaje",
 		"best-cand-score": "por puntaje",
-		"best-type-match": "Best candidate's type match",
-		"best-cand-type-match": "best candidate's types match?",
-		"best-name": "Best candidate's name match",
-		"best-cand-name": "best candidate's name match?",
+		"best-type-match": "Emparejamiento por el tipo del mejor candidato.",
+		"best-cand-type-match": "¿Los tipos del mejor candidato encajan?",
+		"best-name": "Emparejamiento por el nombre del mejor candidato.",
+		"best-cand-name": "¿El nombre del mejor candidato encaja?",
 		"best-edit-dist": "Por distancia de nombre",
 		"best-cand-edit-dist": "Por distancia de nombre",
 		"best-word-sim": "Por semejanza de nombre",
 		"best-cand-word-sim": "Por semejanza de nombre",
 		"best-type": "Por elemento de consulta",
-		"qa-facets": "QA facets",
-		"qa-results": "QA results",
-		"qa-results2": "QA Results",
+		"qa-facets": "Facetas QA",
```

Comment:
It is intended to check the quality of the results from the reconciliation server. Process is the following: 
1. The user send data to a reconciliation server,
2. Reconciliation server returns matches with a score
3. The user can use those facet to navigate the matches and approved / reject them